code/modules/mob/living/carbon/human/species.dm

```diff
@@ -2185,3 +2185,342 @@ GLOBAL_LIST_EMPTY(features_by_species)
  */
 /datum/species/proc/on_owner_login(mob/living/carbon/human/owner)
 	return
+
+/**
+ * Gets a short description for the specices. Should be relatively succinct.
+ * Used in the preference menu.
+ *
+ * Returns a string.
+ */
+/datum/species/proc/get_species_description()
+	return "No description set, yell at a coder!"
```

Comment:
`CRASH` here instead, and unit test it.

Reply:
i hope you realize i have to add monkey lore now otherwise master will spontaneously fail on monkey day